Makefile

```diff
@@ -0,0 +1,61 @@
+.PHONY: tests
+
+all:	clear upfluence yarn_link_ls echo start ## Runs yarn_link_ls & starts the dev server
```

Comment:
It is pertinent to keep command with `start`, we can't really run a server with on `ember-upf-utils`. Same for `re` command